clients/cli/go.sum

```diff
@@ -220,6 +220,8 @@ github.com/pelletier/go-toml v1.8.1 h1:1Nf83orprkJyknT6h7zbuEGUEjcyVlCxSUGTENmNC
 github.com/pelletier/go-toml v1.8.1/go.mod h1:T2/BmBdy8dvIRq1a/8aqjN41wvWlN4lrapLU/GW4pbc=
 github.com/phrase/phrase-go/v2 v2.6.8 h1:PsEUUFE+6gaGPOXxl9L10iwMT/sR94YDGDrQTm41MVo=
 github.com/phrase/phrase-go/v2 v2.6.8/go.mod h1:N1ou4nnB9ak8aAWOH7x25Xhzw8FmFeBRhSGq5A0CY2o=
+github.com/phrase/phrase-go/v2 v2.7.0 h1:T3SgMkl+rciEVh29SvY5PCnubw1PTRf7Lhx3f8bzShs=
```

Comment:
Why is this adding a new version instead of replacing it?

cc @theSoenke

Reply:
https://stackoverflow.com/questions/69825533/why-does-go-sum-include-so-many-older-packages

Reply:
Thanks!

Reply:
`go mod tidy` will also cleanup unused versions. If you run `npm run generate.cli` it was removed for me as it is running the tidy step at the end

Reply:
Updated it, thanks all! 🙌